contributions/catalog/webapps/templates/head.html

```diff
@@ -0,0 +1,10 @@
+<meta content="width=device-width, initial-scale=1" name="viewport">
```

Comment:
I think we need to move this probably to the base template HTML file in a future task. Thanks.

Reply:
same... we have to clean up htmls more...